app/src/main/java/layout/mapFragment.java

```diff
@@ -127,6 +124,13 @@ public void onCreate(Bundle savedInstanceState) {
         //setContentView(R.layout.fragment_map);
 
         //testing get array of moods
+
+        if(!FeelTripApplication.getFrag().equals(frag)) {
+            FeelTripApplication.setFrag(frag); //TODO: Put this wherever we initialize the fragment, I think here works fine.
+        }
```

Comment:
This block of code was in the wrong spot, leading to an incorrect initialization of the map fragment. This has been fixed.